### ISSUE TITLE
old jsonwebtoken version

### ISSUE THREAD

**Issue by kabytaa** [Natashkinsasha/jwt-redis#2]
Please update "jsonwebtoken" version to fix the security isuee.

```
> npm audit

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ hoek                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ jwt-redis                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ jwt-redis > jsonwebtoken > joi > hoek                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/566                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ hoek                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ jwt-redis                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ jwt-redis > jsonwebtoken > joi > topo > hoek                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/566                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 moderate severity vulnerabilities in 4091 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
```

```
> npm list hoek
server@0.0.1 
└─┬ jwt-redis@3.1.0
  └─┬ jsonwebtoken@7.4.3
    └─┬ joi@6.10.1
      ├── hoek@2.16.3
      └─┬ topo@1.1.0
        └── hoek@2.16.3  deduped

```

**Comment by Natashkinsasha** [comment]
Thank you for participating in the development of the project. But I re-wrote the library. Here is a link to repository https://github.com/Natashkinsasha/jwt-redis-v2.git. This version publish in npm and has version 5.*.*

**Comment by Natashkinsasha** [comment]
> Please update "jsonwebtoken" version to fix the security isuee.
> 
> ```
> > npm audit
> 
>                        === npm audit security report ===
> 
> ┌──────────────────────────────────────────────────────────────────────────────┐
> │                                Manual Review                                 │
> │            Some vulnerabilities require your attention to resolve            │
> │                                                                              │
> │         Visit https://go.npm.me/audit-guide for additional guidance          │
> └──────────────────────────────────────────────────────────────────────────────┘
> ┌───────────────┬──────────────────────────────────────────────────────────────┐
> │ Moderate      │ Prototype pollution                                          │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Package       │ hoek                                                         │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Patched in    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                  │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Dependency of │ jwt-redis                                                    │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Path          │ jwt-redis > jsonwebtoken > joi > hoek                        │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ More info     │ https://nodesecurity.io/advisories/566                       │
> └───────────────┴──────────────────────────────────────────────────────────────┘
> ┌───────────────┬──────────────────────────────────────────────────────────────┐
> │ Moderate      │ Prototype pollution                                          │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Package       │ hoek                                                         │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Patched in    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                  │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Dependency of │ jwt-redis                                                    │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Path          │ jwt-redis > jsonwebtoken > joi > topo > hoek                 │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ More info     │ https://nodesecurity.io/advisories/566                       │
> └───────────────┴──────────────────────────────────────────────────────────────┘
> found 2 moderate severity vulnerabilities in 4091 scanned packages
>   2 vulnerabilities require manual review. See the full report for details.
> ```
> 
> ```
> > npm list hoek
> server@0.0.1 
> └─┬ jwt-redis@3.1.0
>   └─┬ jsonwebtoken@7.4.3
>     └─┬ joi@6.10.1
>       ├── hoek@2.16.3
>       └─┬ topo@1.1.0
>         └── hoek@2.16.3  deduped
> ```

Thank you for participating in the development of the project. But I re-wrote the library. Here is a link to repository https://github.com/Natashkinsasha/jwt-redis-v2.git. This version publish in npm and has version 5..
